src/NzbDrone.Core/MediaFiles/ScriptImportDecider.cs

```diff
@@ -25,23 +28,75 @@ public class ImportScriptService : IImportScript
         private readonly IProcessProvider _processProvider;
         private readonly IConfigService _configService;
         private readonly ITagRepository _tagRepository;
+        private readonly IDiskProvider _diskProvider;
         private readonly Logger _logger;
 
         public ImportScriptService(IProcessProvider processProvider,
                                    IVideoFileInfoReader videoFileInfoReader,
                                    IConfigService configService,
                                    IConfigFileProvider configFileProvider,
                                    ITagRepository tagRepository,
+                                   IDiskProvider diskProvider,
                                    Logger logger)
         {
             _processProvider = processProvider;
             _videoFileInfoReader = videoFileInfoReader;
             _configService = configService;
             _configFileProvider = configFileProvider;
             _tagRepository = tagRepository;
+            _diskProvider = diskProvider;
             _logger = logger;
         }
 
+        private static readonly Regex MediaFileLine = new Regex(@"^\[SonarrMediaFile\]\s*(?<mediaFileName>.*)$", RegexOptions.Compiled);
+
+        private static readonly Regex ExtraFileLine = new Regex(@"^\[SonarrExtraFile\]\s*(?<extraFileName>.*)$", RegexOptions.Compiled);
```

Comment:
I think something like:

```suggestion
        private static readonly Regex OutputRegex = new Regex(@"^\[(?:(?<mediaFile>MediaFile)|(?<extraFile>ExtraFile))\]\s(?<fileName>.+)$", RegexOptions.Compiled);
```

for both these regexes to be a little more strict on the format and ensuring the script provided information instead of a null/empty value.

Changed to `\[MediaFile\]` to make it universal with downstream apps and merged into a single regex.